source/data_at_rest_encryption.rst

```diff
@@ -0,0 +1,99 @@
+.. _psmdb.data-at-rest-encryption:
+
+================================================================================
+Data at Rest Encryption
+================================================================================
+
+Data at rest encryption for the WiredTiger storage engine in |mongodb| was
+introduced in |mongodb-enterprise| version 3.2. to ensure that encrypted data
+files can be decrypted and read by parties with the decryption key.
+
+.. seealso::
+
+   |mongodb| Documentation: Encryption at Rest
+       https://docs.mongodb.com/manual/core/security-encryption-at-rest/#encryption-at-rest
+
+.. rubric:: Differences from Upstream
+
+The |feature| in |PSMDB| is introduced in version 3.6 to be compatible with
+|feature| in |mongodb|. In the current release of |PSMDB|, the |feature| does
+not include support for |abbr.kmip|, |vault| or |amazon-aws| key management
+services.
```

Comment:
I think we should add detail to answer these questions:
1. What exactly is and is-not encrypted by this feature?
    - Data files
    - Journal
    - Oplog
    - Log files
2. "Should I expect slower performance?" - I think we should say there is a performance impact to encryption. We don't know the exact impact until we benchmark, so just a general comment is fine.